Sources/ProcedureKit/Delay.swift

```diff
@@ -121,7 +121,7 @@ public class DelayProcedure: Procedure {
         switch delay.interval {
         case (let interval) where interval > 0.0:
             guard !isCancelled else { return }
-            _timer = DispatchSource.makeTimerSource(flags: [], queue: DispatchQueue.default)
+            _timer = eventQueue.makeTimerSource(flags: [])
```

Comment:
Looks like you've got a default argument for `flags`, so the explicit empty set is unnecessary.